### PR TITLE
Post for new newsletter editor

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-editor.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Overall it's a few hours of work each month - except for August and Janary, where we take a little holiday. For more detail, the whole process is documented (of course!) `on the WTD site </organizer-guide/newsletter/newsletter-process/>`__.
+
+You don't have to get started from scratch - if you'd like, you can pair with Beth for a month or two to get into the swing of things.
+
+-----------
+Interested?
+-----------
+
+If you have any questions, or if you think you'd like to be our new editor, drop us a line at newsletter@writethedocs.org!
```
Comment (ericholscher) [inline docs/blog/newsletter-editor.rst]:
```suggestion
If you have any questions, or if you think you'd like to be our new editor, drop us a line at newsletter@writethedocs.org! You can also ping Beth Aitman, or Eric Holscher on Slack to get more information if that's easier. 

```

File: docs/blog/newsletter-editor.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Our outgoing editor (Beth Aitman) has loved being on the team and would highly recommend it. It's a great opportunity to help a wide audience of documentarians, by sharing the knowledge from Slack in a more permanent form; it's also a fun way to practise or experiment with a different style of writing. The format isn't set in stone either - we welcome new ideas for how to make the newsletter even better.
+
+--------------
+What you'll do
+--------------
+
+You'll be responsible for putting together the newsletter each month. The process looks roughly like this:
+
+- During the month, keep an eye on Slack and flag any interesting conversations.
```
Comment (ericholscher) [inline docs/blog/newsletter-editor.rst]:
```suggestion
- During the month, keep an eye on Slack and flag any interesting conversations. The community also able to flag conversations for the newsletter, which you can draw from. 
```

**Review by ericholscher — APPROVED** [review]
This looks great. A couple additions/suggestions.

File: docs/blog/newsletter-editor.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Towards the end of the month, pick 4-5 topics to turn into articles.
+- Share the topics to be written up with our awesome team of writers, and write one yourself.
+- Edit the articles into a coherent newsletter and fill out the rest (news, jobs, meetup) .
+
+Overall it's a few hours of work each month - except for August and Janary, where we take a little holiday. For more detail, the whole process is documented (of course!) `on the WTD site </organizer-guide/newsletter/newsletter-process/>`__.
```
Comment (ericholscher) [inline docs/blog/newsletter-editor.rst]:
```suggestion
Overall it's a few hours of work each month - except for August and Janary, where we take a little holiday. For more detail, the whole process is documented (of course!) `on the WTD site </organizer-guide/newsletter/newsletter-process/>`__. This role has an optional stipend of $200/mo to help ensure that you can dedicate time to completing the newsletter.
```

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
~I'm not sure why this isn't running~

Oh I see, another incident.

**Comment by plaindocs** [comment]
Something broken with the build statuses on this, so I'm just gonna merge.

**Pull request merged**